neuro_extras/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 
 # NOTE: When updating the version, don't forget to update main.NEURO_EXTRAS_IMAGE_TAG
-__version__ = "20.9.25"
+__version__ = "v20.9.29a2"
```

Comment:
we keep the version here as `YY.MM.DD` but release on GitHub as `vYY.MM.DD`
But it's a good decision to follow a consistent approach.
@mariyadavydova, which one should we follow?